firebase-installations/src/main/java/com/google/firebase/installations/FirebaseInstallationsException.java

```diff
@@ -20,11 +20,12 @@
 /** The class for all Exceptions thrown by {@link FirebaseInstallations}. */
 public class FirebaseInstallationsException extends FirebaseException {
 
-  // TODO(ankitagj): Improve clear exception handling.
   public enum Status {
```

Comment:
I think we need clear javadoc for those enum, what do they mean and what does developer need to do,

Reply:
Added a TODO.